templates/firefox-ios.xliff

```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
+        <source>Unfortunately, we can’t check the review quality for certain types of products. For example, gift cards and streaming video, music, and games.</source>
+        <note>Title for info card when Fakespot cannot analyze reviews for a certain product type</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.FakespotDoesNotAnalyzeReviews.Title.v120" xml:space="preserve">
+        <source>Can’t Check These Reviews</source>
+        <note>Title for info card when Fakespot cannot analyze reviews for a certain product type</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NeedsAnalysis.PrimaryAction.v120" xml:space="preserve">
+        <source>Check Now</source>
+        <note>Primary action title for info card when the product needs analysis</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NeedsAnalysis.Title.v120" xml:space="preserve">
+        <source>New Info To Check</source>
+        <note>Title for info card when the product needs analysis</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NoInfoAvailableRightNow.Description.v120" xml:space="preserve">
+        <source>We’re working to resolve this issue. Please check back soon.</source>
+        <note>Description for info card when no information is available at the moment</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NoInfoAvailableRightNow.Title.v120" xml:space="preserve">
+        <source>No Info Available Right Now</source>
+        <note>Title for info card when no information is available at the moment</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NotEnoughReviews.Description.v120" xml:space="preserve">
+        <source>When this product has more reviews, we’ll be able to check their quality.</source>
+        <note>Description for info card when there are not enough reviews for a product</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NotEnoughReviews.Title.v120" xml:space="preserve">
+        <source>Not Enough Reviews Yet</source>
+        <note>Title for info card when there are not enough reviews for a product</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.ProgressAnalysis.Description.v120" xml:space="preserve">
+        <source>This could take about 60 seconds.</source>
+        <note>Description for info card when the product is in analysis mode</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.ProgressAnalysis.Title.v120" xml:space="preserve">
+        <source>Checking review quality</source>
+        <note>Title for info card when the product is in analysis mode</note>
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.AnalyzerButton.Title.v120" xml:space="preserve">
+        <source>Check Review Quality</source>
+        <note>Text for the analyzer button, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
```

Comment:
The comment might be a bit confusing to localizers and seems more dev oriented, from this part "from No Analysis Card View displayed in the shopping review quality bottom sheet". 
Something more like "text for the analyzer button, when the analysis can be updated for a product" (or something along those lines, if there's a better description)

Reply:
Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940